### ISSUE TITLE
Dynamis - Xarcabard Animated horn not meleeable

### ISSUE THREAD

**Issue by ehose** [EdenServer/community#5813]
### Checklist

- [x ] I have searched for existing issues for issues like this one. The issue has not been posted.
- [ ] I have provided reproducible steps. (No "as the title says" posts please. ***Please try to state facts only, and avoid assumptions/opinions.***)
- [x ] I have provided my client version in the details. (type /ver into your game window)
- [ x] ***I have provided proof of my claim (era videos, links to era wiki and/or forum posts)*** Lack of proof will slow down fixes and prioritization. Please provide details so that developers' research time is minimized.

### Details

Fighting the Animated Horn in Dynamis - Xarcabard with melee resaults in not being able to see/hit message once it is on the ground, we have tried to fight it during two different runs and received the same message. Ranged attacks do land on the Animated Horn though. 
client version: 30181205_0
Picture of message: 
![image](https://github.com/EdenServer/community/assets/150992495/93d28c27-b592-4639-b207-4e368624bbc2)

Era video of it being meleed:
https://www.youtube.com/watch?v=-TPejraQizk&ab_channel=scp187

**Comment by Mozzy4Ever** [comment]
For the record: It is meleeable. Best to let it come completely off the wall and path a little bit to a flatter area. Does have a tendency to get in a bugged spot like other mobs.

**Comment by eartsar** [comment]
This is the second attempt wherein this has happened. It also exhibits questionable pathing behavior on the ledge. 

It seems that downing this requires funky strategies or finicky pathing work-arounds to deal with, both of which eat into the encounter time, _which has an outsized impact on animated weapon encounters_ specifically. In the case of _this_ animated weapon, perhaps we're unlucky, but it certainly feels suspect that its spawn location on the ledge, and the uneven elevation of the landing, leaves this mob even more prone to these issues.

Ultimately, after an unlucky song cast on the ledge, and funky pathing to come down, there remained something around 12 seconds left of encounter time. That's not a lot of time to finagle!

Our next attempt will do something unconventional, as in-era strategies to deal with mob are hit-or-miss here. Short of a systematic change on whatever is causing z-axis movement to be so funky on Eden, it might be worth considering adjusting the warp window, spawn location, or having some recourse process to abate the frustration on the animated weapons specifically.

**Comment by Mozzy4Ever** [comment]
We've done it twice by treating it like the other animated weapon fights. THF w/ Flee RA's it and runs away. Everyone waits for it to come down from the ledge and out of it's little alcove, then engages.

**Comment by eartsar** [comment]
Yup. Same strategy. This era tried-and-true strategy is extremely simple and obvious. I don't think this bug ticket is indicating the encounter as impossible to clear, of course.

Our group simply got unlucky twice. The root cause is probably more tricky to address, and are well known quirks on this private server with regards to positioning. But taking a few seconds to reposition a monster, one that's a caster and prone to wasting time in those compromised positions, for a fight that lasts less than 30 seconds, is perhaps a little _too_ prone to bad luck, and the bad luck is perhaps more impactful here than it would be in any other type of encounter.

**Comment by DeuceFFXI** [comment]
This is ultimately a navmesh issue that affects mobs in various zones I believe.  The navmesh and/or pathing logic needs to be worked on as a whole to fix this sort of issue.  I'm sure others have noticed mobs on uneven terrain cause players to have this same issue of "out of range" when they're right next to the mob.
